contracts/oracles/TwapOracle.sol

```diff
@@ -173,25 +170,25 @@ contract TwapOracle is OwnableUpgradeable, OracleInterface {
 
         // This should be impossible, but better safe than sorry
         require(block.timestamp > oldTimestamp, "now must come after before");
-        uint256 timeElapsed = block.timestamp.sub(oldTimestamp);
+        uint256 timeElapsed = block.timestamp - oldTimestamp;
 
         // Calculate Pancakge TWAP
         FixedPoint.uq112x112 memory priceAverage = FixedPoint.uq112x112(uint224(
-            nowCumulativePrice.sub(oldCumulativePrice).div(timeElapsed)
+            (nowCumulativePrice - oldCumulativePrice) / timeElapsed
         ));
         // TWAP price with 1e18 decimal mantissa
         uint256 priceAverageMantissa = priceAverage.decode112with18();
 
         // To cancel the decimals in cumulative price, we need to mulitply the average price with 
         // tokenBaseUnit / (wbnbBaseUnit or busdBaseUnit, which is 1e18)
         uint256 pairedTokenBaseUnit = config.isBnbBased ? bnbBaseUnit : busdBaseUnit;
-        uint256 anchorPriceMantissa = priceAverageMantissa.mul(config.baseUnit).div(pairedTokenBaseUnit);
+        uint256 anchorPriceMantissa = (priceAverageMantissa * config.baseUnit)/pairedTokenBaseUnit;
```

Comment:
being picky: there is no spaces around `/` while this PR introduces them in all other places

Reply:
Since we use `solhint` we should leverage that for auto formatting on commit and checking in CI